src/render/render.js

```diff
@@ -50,6 +54,8 @@ module.exports = (page, metaIndex) => {
       throw new Error(`Failed to find tag structure for ${page.title}`);
     }
 
+    keywords.push(tag.id);
```

Comment:
If the page is a tag page, the engine ID is a search keyword (e.g. search "dela" and get suggested `ui_widget_definition`)